### PR TITLE
Fix #13: Import button is always visible in Manager Modal

### LINKED ISSUE #13: Import button is always visible in Manager Modal
Opposing to the documentation:
```javascript
modules: {
  'apostrophe-pieces-import': {},
  // other modules...
  'my-module-that-extends-pieces': {
    // Without this, there is no import functionality for this type.
    // Not all types are great candidates for imports.
    import: true
  }
}
```
The import button is always visible.

### PR DESCRIPTION
**Pull request by casabian**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -22,3 +22,6 @@ test/public/css/master-*.less
 
 # Dont commit test uploads
 test/public/uploads
+
+# Dont commit vscode
```
Comment (boutell) [inline .gitignore]: Please remove this from the PR.

File: index.js [hunk 2]
```diff
@@ -160,7 +158,7 @@ module.exports = {
 
         // Do the real work, without worrying about the browser hanging up
 
-        return async.series([ insertAndSniff, count, storeCount ], function(err) {
+        return async.series([insertAndSniff, count, storeCount], function(err) {
```
Comment (boutell) [inline index.js]: Bring back the whitespace.

File: index.js [hunk 3]
```diff
@@ -296,9 +293,8 @@ module.exports = {
     // invoked at all in the event of a cancellation.
 
     self.importCancelOrContinue = function(job, callback) {
-
       return self.db.findOne({ _id: job._id }, function(err, _job) {
-        if (err || (!_job)) {
+        if (err || !_job) {
```
Comment (boutell) [inline index.js]: Bring back the parens.

File: index.js [hunk 4]
```diff
@@ -316,7 +312,7 @@ module.exports = {
           job.parser.end();
         }
 
-        return async.series([ waitForLast, markCanceled, remove ]);
+        return async.series([waitForLast, markCanceled, remove]);
```
Comment (boutell) [inline index.js]: Bring back the whitespace - please do not alter the style.

File: index.js [hunk 5]
```diff
@@ -383,16 +377,20 @@ module.exports = {
...
-          if (err) {
-            return callback(err);
-          }
-          if (!existing) {
-            return callback('update-notfound ' + query[keyField]);
-          }
-          piece = existing;
-          return callback(null);
-        });
+        return self
```
Comment (boutell) [inline index.js]: Too many style changes to locate your real changes, please resubmit without these. Thanks.

**Review by boutell — CHANGES_REQUESTED** [review]
Sorry for the hassle, but I need you to resubmit without the style changes elsewhere in the file that make it difficult to be sure I've tracked down all the actual semantic changes. Thanks!

File: index.js (hunk 5 above)
Comment (casabian) [reply index.js]: Pushed version without style changes!

**Review by boutell — APPROVED** [review]

**Comment by boutell** [comment]
Thanks!

**Comment by boutell** [comment]
This shipped as version 2.1.5.

**Pull request merged**